package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyux/pages",
-  "version": "0.0.1",
+  "version": "4.0.0b0",
```

Comment:
We'll want this to be `4.0.0-beta.0`.

We have some pipeline things that look for that specific beta annotation.